### PR TITLE
placeholder stays gray now

### PR DESCRIPTION
**Pull request by billyfung**
#77 placeholder text stays gray when in focus

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -53,3 +53,24 @@ footer.ui.footer {
...
+  border-color: #85B7D9;
+  background: #FFFFFF;
+  color: rgba(0, 0, 0, 0.8);
+  box-shadow: none;
+}
+.ui.input.focus input::-webkit-input-placeholder,
+.ui.input input:focus::-webkit-input-placeholder {
+  color: rgba(128, 128, 128, 1);
+}
+.ui.input.focus input::-moz-placeholder,
```
Comment (jinpark) [inline static/style.css:61]: you can probably merge all this into one css rule?

Comment (billyfung) [inline static/style.css:61]: probably, I just copy and pasted all that from the source, wasn't sure what it does

Comment (jinpark) [inline static/style.css:61]: ...

Comment (billyfung) [inline static/style.css:61]: how do I merge it all into one cs rule?

Comment (jinpark) [inline static/style.css:61]: http://www.w3schools.com/css/css_syntax.asp

look at grouping selectors

Comment (billyfung) [inline static/style.css:61]: but I thought you can't group selectors if one if them is invalid?

Comment (jinpark) [inline static/style.css:61]: Try it and see :D

Comment (billyfung) [inline static/style.css:61]: doesn't seem to work for me

**Comment by jinpark** [comment]
LGTM

I learned more about the placeholders too :D https://css-tricks.com/almanac/selectors/p/placeholder/

**Pull request merged**